### PR TITLE
Create and package readonly node.

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**
This disables all write request from the node, both on protocol level and on the storage engine level, and package it using the new user, directory, and upstart scripts.

### CODE REVIEW

File: src/dlsnode/neo/request/Put.d [hunk 1]
```diff
@@ -139,14 +139,8 @@ private scope class PutImpl_v0: PutProtocol_v0
 
     override protected bool prepareChannel ( cstring channel_name )
     {
```
Comment (gavin-norman-sociomantic) [inline src/dlsnode/neo/request/Put.d]: Actually, it's probably better to just remove these requests from the map of request handlers. That way, the neo protocol will automatically send back a "not supported" status code to the client.

**Comment by nemanja-boric-sociomantic** [comment]
Updated and unblocked.

**Comment by gavin-norman-sociomantic** [comment]
Does the last commit undo what a previous commit did? Squash?

**Comment by nemanja-boric-sociomantic** [comment]
Squashed.

**Comment by gavin-norman-sociomantic** [comment]
Feel free to auto-merge when ready.

**Pull request merged**